src/redux/migrations.test.ts

```diff
@@ -1584,7 +1584,7 @@ describe('Redux persist migrations', () => {
     const oldSchema = {
       ...v203Schema,
       positions: {
-        ...v203Schema.positions,
+        ...(v203Schema.positions as any),
```

Comment:
these `any` casts became necessary after the changes in the schemas.ts file...we create schemas to migrate from old data types that we don't keep anymore so casting the old data types as `any` seems to make sense